src/redux/actions/Dashboard/index.js

```diff
@@ -0,0 +1,38 @@
+import {
+    getDashboardTotals,
+    getDashboardChartsData,
+    getDashboardLastMessages,
+    getDashboardNewUsers,
+} from '../../services/Dashboard';
+
+export const LOADING_DASHBOARD_DATA = 'LOADING_DASHBOARD_DATA';
+export const STOP_LOADING_DASHBOARD_DATA = 'STOP_LOADING_DASHBOARD_DATA';
+export const LOAD_DASHBOARD_DATA = 'LOAD_DASHBOARD_DATA';
+
+export function fetchDashboardData() {
+    return (dispatch) => {
+        dispatch({ type: LOADING_DASHBOARD_DATA });
```

Comment:
before all this you need to check if is first time in order to not fetch data again when the user come back to the page